src/communication/xcWebSocketSubscriber.js

```diff
@@ -8,7 +8,7 @@ define(["rx"], function (Rx) {
 	}
 
 
-	Subscriber.prototype.getEventToSend = function (componentName, stateMachineName) {
+	Subscriber.prototype.getjsonDataToSendSusbcribeRequest = function (componentName, stateMachineName) {
```

Comment:
Important to notice: you don't need to unit test private methods. Just test public methods.
Avoid defining a method as public just for unit tests